templates/nfs/v1/nfs.jinja

```diff
@@ -0,0 +1,50 @@
+{% set PROPERTIES = properties or {} %}
+{% set SERVER_IP = PROPERTIES['ip'] or '10.0.253.247' %}
+{% set SERVER_PORT = PROPERTIES['port'] or 2049 %}
+{% set SERVER_DISK = PROPERTIES['disk'] or 'nfs-disk' %}
+{% set SERVER_DISK_FSTYPE = PROPERTIES['fstype'] or 'ext4' %}
+{% set CLAIM_NAME = PROPERTIES['claim-name'] or 'nfs' %}
+
+resources:
+- name: nfs
+  type: https://raw.githubusercontent.com/leendersr/deployment-manager/master/templates/replicatedservice/v2/replicatedservice.py
```

Comment:
Is this right? I think it needs to be `type: https://raw.githubusercontent.com/kubernetes/deployment-manager/master/templates/replicatedservice/v2/replicatedservice.py` like rest (as this no longer seems to resolve).